### PR TITLE
Script for bumping the SDK projects versions

### PR DESCRIPTION
**Pull request by wojta**
## Motivation
We need to have some automation for releasing the new SDK versions.

JIRA: https://issues.jboss.org/browse/AEROGEAR-2693

## Description

Create script to increment version number. Configurable for major, minor, patch version number.

## Progress

- [x] Write the version bumping script
- [x] Script to call `nuget pack` and `nuget push`
- [x] Basic test to check if the script works
- [x] Document the task

## Additional Notes

Based on the same approach as the script for modifying Showcase app.

### CODE REVIEW

File: scripts/update_showcase.js [hunk 1]
```diff
@@ -7,10 +7,10 @@ Script for updating Showcase Template App from the SDK Example app.
 
 const yargs = require("yargs")
```
Comment (ziccardi) [inline scripts/update_showcase.js]: If you run this script multiple times, it adds the dependencies to the projects multiple times, thus breaking the compilation.

Comment (wojta) [reply scripts/update_showcase.js]: This is fixed now. Please review again.

File: scripts/modules/sdk_operations.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                    splitVersion[0]++
+                    splitVersion[1] = 0
+                    splitVersion[2] = 0
+                    break;
+                case 'minor':
+                    splitVersion[1]++
+                    splitVersion[2] = 0
+                    break;
+                case 'patch':
+                    splitVersion[2]++
```
Comment (wei-lee) [inline scripts/modules/sdk_operations.js]: how does this work if the version has labels like `1.0.0-alpha`? There is a semver package in npm, might be better using that instead?

File: scripts/showcase_config.json [hunk 3]
```diff
@@ -2,30 +2,46 @@
...
+            "AeroGear.Mobile.Security.Platform.Android": "1.0.0-alpha"
         },
         "Example/Example.iOS/Example.iOS.csproj": {
-            "AeroGear.Mobile.Auth.Platform.iOS": "0.0.6"
+            "AeroGear.Mobile.Core": "1.0.0-alpha",
+            "AeroGear.Mobile.Core.Platform.iOS": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth": "1.0.0-alpha",
+            "AeroGear.Mobile.Auth.Platform.iOS": "1.0.0-alpha",
+            "AeroGear.Mobile.Security": "1.0.0-alpha",
+            "AeroGear.Mobile.Security.Platform.iOS": "1.0.0-alpha"
```
Comment (wei-lee) [inline scripts/showcase_config.json]: should this script be part of the showcase app repo?

Comment (wojta) [reply scripts/showcase_config.json]: I had it there before. But I decided that this is better place. Script is used only for the development. Also it shares some code with the SDK release script.

**Pull request merged**